package.json

```diff
@@ -13,13 +20,13 @@
     "query",
     "json"
   ],
+  "scripts": {
+    "start": "node ./example/server.js"
+  },
   "dependencies": {
-    "json-mask": "0.3.2"
+    "json-mask": "0.3.8"
   },
-  "author": "nemtsov@gmail.com",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/nemtsov/express-partial-response.git"
+  "devDependencies": {
+    "express": "^4.16.2"
```

Comment:
How do you feel about adding a `package.json` inside the example directory instead and adding `example/node_modules` to `.gitignore` (maybe along with `package-lock.json` [unrelated]). This way it's clear that that's the dependency of the example and not of the main package.